data_changes.md

```diff
@@ -0,0 +1,58 @@
+## Changes to Imported Data ##
+
+Generally, data will be imported as-is from historical data or newly given data
+with a few exceptions:
+
+- _IMPORT TYPE_: Broker, USASpending
+- _CHANGE_: **Creation of summary awards**
```

Comment:
I'm torn about whether or not we should include info re: summary award. It's a bit different than "here's some incoming data that we changed." If we do put info about award creation into this version of the document, I think we need to elaborate a bit more about how we create these awards and watch for trigger words like "summary" and "parent awards" that will confuse agency folks who are already familiar with the domain.

Happy to work on wording if that's helpful.